examples/browser-video-streaming/index.html

```diff
@@ -2,7 +2,11 @@
   <body>
     <video id="video" controls></video>
     <script src="../../dist/index.js"></script>
+    <!--
+      TODO: uncomment once https://github.com/ipfs/js-ipfs/pull/2683 and
+      https://github.com/moshisushi/hlsjs-ipfs-loader/pull/16 are released
     <script src="https://unpkg.com/hlsjs-ipfs-loader@0.1.4/dist/index.js"></script>
+    -->
```

Comment:
Can now be uncommented?

Reply:
See the other comment, the `hlsjs-ipfs-loader` PR hasn't been published yet